Altinn.Platform.Authentication.sln

```diff
@@ -13,6 +13,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Altinn.Platform.Authenticat
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Altinn.Platform.Authentication.Integration", "src\Integration\Altinn.Platform.Authentication.Integration.csproj", "{A367652C-B409-40DE-8ACA-C65FF1107E78}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Altinn.Platform.Authentication.SystemIntegrationTests", "Altinn.Platform.Authentication.SystemIntegrationTests\Altinn.Platform.Authentication.SystemIntegrationTests.csproj", "{A1228D39-231E-4301-82D2-0524F49A01EB}"
```

Comment:
Er det noen grunn til at dette prosjektet skal ligge i denne solution? 
Trodde det økte risikoen for at man gjenbrukte modeller og annet på tvers?

Reply:
Var det jeg noterte meg etter den første praten vi hadde rundt disse testene. 
Mine argumenter er hvert fall:
- Senke terskelen for at utvikler selv får kjørt, feilretter og skrevet tester ved behov (bedre tilgjengelighet)
- Om man låner eller stjeler DTOer så er det bedre enn at man overser testene